### PR TITLE
Added a button on test app to raise an exception from NSApplication

### PR DESCRIPTION
**Pull request by ilyashumihin**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Add a button on test app to raise an exception from NSApplication to Sasquatch project

## Related PRs or issues

[AB#74006](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/74006)

### CODE REVIEW

File: CrashLib/CrashLib/MSCrashNSApp.m [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSCrashNSApp.h"
+
+@implementation MSCrashNSApp
```
Comment (MatkovIvan) [inline CrashLib/CrashLib/MSCrashNSApp.m]:
```suggestion
@implementation MSCrashDelayedObjCException
```

File: CrashLib/CrashLib/MSCrashNSApp.m [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+- (NSString *)title {
+    return @"Throw Objective-C exception avoiding send event";
+}
+
+- (NSString *)desc {
+    return @""
+    "Throw an uncaught Objective-C exception avoiding send event. "
+    "It's possible to generate a better crash report here compared to the C++ Exception case "
```
Comment (MatkovIvan) [inline CrashLib/CrashLib/MSCrashNSApp.m]: Do we need to copy this here?

File: CrashLib/CrashLib/MSCrashNSApp.m [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#import "MSCrashNSApp.h"
+
+@implementation MSCrashNSApp
+
+- (NSString *)category {
+    return @"Exceptions";
+}
+
+- (NSString *)title {
+    return @"Throw Objective-C exception avoiding send event";
```
Comment (MatkovIvan) [inline CrashLib/CrashLib/MSCrashNSApp.m]:
```suggestion
    return @"Throw Objective-C exception outside of IBAction";
```

File: CrashLib/CrashLib/MSCrashNSApp.m (hunk 2 above)
Comment (ilyashumihin) [reply CrashLib/CrashLib/MSCrashNSApp.m]: Resolved

**Pull request merged**